sdk/tests/conformance/extensions/ext-sRGB.html

```diff
@@ -125,9 +125,7 @@
   var err = gl.getError();
   if (err == gl.NO_ERROR) {
     testFailed("should NOT be able to create type " + formatName);
-  } else if (err == gl.INVALID_OPERATION) {
-    testFailed("should return gl.INVALID_ENUM for type " + formatName);
-  } else if (err == gl.INVALID_ENUM) {
+  } else if (err == gl.INVALID_ENUM || err == gl.INVALID_OPERATION) {
```

Comment:
This change also looks incorrect. Per https://www.khronos.org/opengles/sdk/docs/man/xhtml/glTexImage2D.xml the only error condition we should be hitting is that for INVALID_ENUM, since the format isn't an accepted value.